tools/ang_quad_plotting/ANG_QUAD_PLOTTING_README.md

```diff
@@ -0,0 +1,14 @@
+# Plotting Angular Quadrature Discretization using a simple python script
+
+Unfortunately no tutorials exist at this time. A guide follows below for instructions.
+
+When running a problem that manipulates angular quadratures. Assuming the file base is "TestQuad_" the following 3 files will be created:
+TestQuad_verts.csv
+TestQuad_cells.csv
+TestQuad_points.csv
+
+1. Move all three of these files into this folder
+2. In [plot_quadrature.py](plot_quadrature.py) edit line 8 to be the file base indicated above
```

Comment:
all of this should be in a subsection (use double pond) dedicated to SLDFE quadrature plotting. That plotter does not work for other quadratures.

also add two images of the typical plots. one with uniform refinement, one with local refinement.